package/yast2-cluster.spec

```diff
@@ -36,6 +36,7 @@ BuildRequires:  firewall-macros
 BuildArch:      noarch
 
 # SuSEFirewall2 replaced by Firewalld (fate#323460)
+Requires:       sharutils
```

Comment:
Just a note, Ruby has a built-in support for UU encoding and decoding, you do not have to depend on external tools:

```
# irb
irb(main):001:0> ["plain text"].pack("u")
=> "*<&QA:6X@=&5X=```\n"
irb(main):002:0> "*<&QA:6X@=&5X=```\n".unpack("u")
=> ["plain text"]
```

See https://ruby-doc.org/core-2.5.3/Array.html#method-i-pack and https://ruby-doc.org/core-2.5.3/String.html#method-i-unpack.

But as this would require quite a big change in the code I'm fine with this solution for the maintenance update.